packages/utils/src/__tests__/getAccountListWithAuthToken.spec.ts

```diff
@@ -0,0 +1,150 @@
+                    session_start: 1714715763,
+                    excluded_until: '',
+                    landing_company_name: 'svg',
+                    balance: 10000,
+                    accepted_bch: 0,
+                    residence: 'aq',
+                },
+                CRW1122: {
+                    account_category: 'wallet',
+                    account_type: 'crypto',
+                    broker: 'CRW',
+                    created_at: 1714662788,
+                    currency: 'BTC',
+                    currency_type: 'crypto',
+                    is_disabled: 0,
+                    is_virtual: 0,
+                    landing_company_shortcode: 'svg',
+                    linked_to: [],
+                    token: 'xob1xob1xob1xob1xob1xob1xob1',
+                    excluded_until: '',
+                    landing_company_name: 'svg',
+                    balance: 0,
+                },
+                VRW1069: {
+                    account_category: 'wallet',
+                    account_type: 'virtual',
+                    broker: 'VRW',
+                    created_at: 1714662434,
+                    currency: 'USD',
+                    currency_type: 'fiat',
+                    is_disabled: 0,
+                    is_virtual: 1,
+                    landing_company_shortcode: 'virtual',
+                    linked_to: [],
+                    token: 'xob1xob1xob1xob1xob1xob1xob1',
+                    excluded_until: '',
+                    landing_company_name: 'virtual',
+                    balance: 10000,
+                },
+            })
+        );
+
+        accounts = [
+            {
+                account_category: 'wallet',
+                account_type: 'doughflow',
+                broker: 'CRW',
+                created_at: 1714662434,
+                currency: 'USD',
+                currency_type: 'fiat',
+                is_disabled: 0,
+                is_virtual: 0,
+                landing_company_name: 'svg',
+                linked_to: [],
+                loginid: 'CRW1121',
+            },
+            {
+                account_category: 'wallet',
+                account_type: 'crypto',
+                broker: 'CRW',
+                created_at: 1714662788,
+                currency: 'BTC',
+                currency_type: 'crypto',
+                is_disabled: 0,
+                is_virtual: 0,
+                landing_company_name: 'svg',
+                linked_to: [],
+                loginid: 'CRW1122',
+            },
+            {
+                account_category: 'wallet',
+                account_type: 'virtual',
+                broker: 'VRW',
+                created_at: 1714662434,
+                currency: 'USD',
+                currency_type: 'fiat',
+                is_disabled: 0,
+                is_virtual: 1,
+                landing_company_name: 'virtual',
+                linked_to: [],
+                loginid: 'VRW1069',
+            },
```

Comment:
these accounts' data is repeatedly used in one of the tests and when you set it into localstorage above, they can be declared on top and reused using spread operator `...` instead of copying the data again and again :)